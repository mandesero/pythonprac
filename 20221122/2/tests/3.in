Â êà÷åñòâå ïîñëåäíåãî ïðîòèâîÿäèÿ ïðåäïðèíèìàåòñÿ ïîïûòêà èçìåíèòü àòðèáóò family. Çàìåíà êîäèðîâêè áûëà áû ÷ðåâàòà èñêàæåíèåì ñìûñëà òåêñòà èç-çà ïîÿâëåíèÿ ¾íå òåõ¿ áóêâ. Òàêîâ òèïè÷íûé ñöåíàðèé äåéñòâèéNFSS, íî è îí ìîæåò áûòü èçìåí¼í.Ãëàâà 16. Øðèôòû äëÿ ïðîôåññèîíàëîâ386Íàïðèìåð, åñëè çàïðîñèòü ðóêîïèñíûé øðèôò â ïðÿìîì íà÷åðòàíèè íîðìàëüíîéíàñûùåííîñòè (æèðíîãî ðóêîïèñíîãî øðèôòà âñ¼ ðàâíî íåò), òî NFSS, äåéñòâóÿ ïîñòàíäàðòíîé ñõåìå, ïîäìåíèò åãî øðèôòîì äðóãîãî ñåìåéñòâà.
