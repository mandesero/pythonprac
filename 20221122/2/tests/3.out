В качестве последнего противоядия предпринимается попытка изменить атрибут family. Замена кодировки была бы чревата искажением смысла текста из-за появления ѕне техї букв. Таков типичный сценарий действийNFSS, но и он может быть изменјн.Глава 16. Шрифты для профессионалов386Например, если запросить рукописный шрифт в прямом начертании нормальнойнасыщенности (жирного рукописного шрифта всј равно нет), то NFSS, действуя постандартной схеме, подменит его шрифтом другого семейства.

Действительно, поскольку прямое начертание это начертание по умолчанию, равно как и нормальнаянасыщенность есть насыщенность по умолчанию, NFSS сразу переключит атрибут семейства. Разработчики шрифтового пакета могут задать иные правила подмены (всј втом же файле определения шрифтов или в самом пакете) с тем, чтобы предотвратитьподмену гарнитуры:{\usefont{T1}{pzc}{m}{it} ZapfChancery}\\{\usefont{T1}{pzc}{m}{n} ZapfChancery}ZapfChanceryZapfChanceryБольшинство операций со шрифтами, производимые в процессе компиляции документа, фиксируются в файле протокола (с расширением log). Расшифровка этихсообщений приведена в приложении B.

Подробность сообщений можно регулировать,если загрузить пакет tracefnt с различными опциями. Вот эти опции, перечисленныев порядке возрастания подробности сообщений: errorshow, warningshow, infoshow,debugshow, loading, pausing.16.8.Пакеты PSNFSSНачало коллекции PSNFSS было положено Себастьяном Ратцем (Rahtz, Sebastian).Наряду с поддержкой упоминавшегося в начале главы набора 35 PostScript-шрифтовфирмы Adobe коллекция PSNFSS содержала пакеты для подключения множества других PostScript-шрифтов, которые большей частью необходимо приобретать за отдельную плату.

В настоящее время поддержка коммерческих шрифтов выделена в самостоятельную коллекцию PSNFSSx. Среди пакетов PSNFSSx следует выделить пакетыhvmaths, lucidabr, ly1, mathtime, mtpro, tmmaths, которые предназначены для заменыматематических шрифтов Computer Modern на современные шрифты Lucida Bright иMathTime, продаваемые компаниями Y&Y 9 и MicroPress10 . Мы сосредоточим внимание Читателя на некоммерческих шрифтах, поддерживаемых пакетами PSNFSS, темболее что здесь также есть замечательные шрифты семейств Palatino и Times для математических формул.

В настоящее время пакеты PSNFSS сопровождает Вальтер Шмидт(Schmidt, Walter).Установка всего необходимого для работы с пакетами PSNFSS, включая шрифты,может потребовать изучения сопроводительной документации. Мы можем только засвидетельствовать, что в полном комплекте библиотеки MiKTeX оказалось всј, чтонужно, но в ѕразобранном видеї, а именно: собственно коллекция PSNFSS, шрифты Courier, Utopia компании Adobe, Charter компании Bitstream, PazoMath Диего Пуги(Puga, Diego) находились в разных архивах. Что касается 35 шрифтов Adobe, то ихбесплатный клон входит в состав программы Ghostscript, которая является почти обязательной составной частью любой современной реализации системы LATEX.